src/utils.js

```diff
@@ -572,7 +572,16 @@ export default {
       let i
       let nu
 
-      if (typeof value === 'object' && value !== null && !(value instanceof Boolean) && !(value instanceof Date) && !(value instanceof Number) && !(value instanceof RegExp) && !(value instanceof String) && !(value instanceof File)) {
+      if (
+        typeof value === 'object' &&
+        value !== null &&
+        !(value instanceof Boolean) &&
+        !(value instanceof Date) &&
+        !(value instanceof Number) &&
+        !(value instanceof RegExp) &&
+        !(value instanceof String) &&
+        (typeof File === 'undefined' || !(value instanceof File))
```

Comment:
Fixed, although I just noticed [this](https://github.com/js-data/js-data/blob/master/src/utils.js#L40) which is the actual cause of the bug (it will always be undefined in IE9 rather than throwing an error because you're accessing it on the window variable). I think this should just be removed and my check kept in place, as shadowing global variables in JS isn't good practise and can cause weird behaviour in bad JS engines like IE. What do you think?